src/Function/Construct/Symmetry.agda

```diff
@@ -35,21 +37,26 @@ module _ {≈₁ : Rel A ℓ₁} {≈₂ : Rel B ℓ₂} {f : A → B}
          ((inj , surj) : Bijective ≈₁ ≈₂ f)
          where
 
-  private
-    f⁻¹      = proj₁ ∘ surj
-    f∘f⁻¹≡id = proj₂ ∘ surj
+  open IsSurjective {≈₂ = ≈₂} surj
+    using (section; section-strictInverseˡ)
+
+  module _ (refl : Reflexive ≈₁) where
+
+    private
+
+      f∘section≡id = section-strictInverseˡ refl
```

Comment:
This seems to be a large increase in both lines and complexity of code for comparatively little benefit down below?

Reply:
This refers to an out-of-date commit now?